flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionLoader.java

```diff
@@ -154,8 +169,69 @@ private void moveFiles(List<Path> srcDirs, Path destDir) throws Exception {
         }
     }
 
+    private void createSuccessFile(LinkedHashMap<String, String> partitionSpec, Path path)
+            throws Exception {
+        if (writeSuccessFileEnabled) {
+            PartitionCommitPolicy.Context context =
+                    new CommitPolicyContextImpl(partitionSpec, path);
+            commitPolicy.commit(context);
+        }
+    }
+
     @Override
     public void close() throws IOException {
         metaStore.close();
     }
+
+    private class CommitPolicyContextImpl implements PartitionCommitPolicy.Context {
```

Comment:
Add java doc for this class, and explain it's only for write success file  currently, so that only the method  `partitionPath`, `partitionSpec` is needed.